### PR TITLE
Processing updates

### PR DESCRIPTION
**Pull request by simonpf**
Two fixes for the processing pipeline:
1. Wrap file download in try/except to avoid stalling processing
2. Only add altitude coordinates when results contain profiles

### CODE REVIEW

File: ccic/bin/process.py [hunk 1]
```diff
@@ -315,7 +315,13 @@ def download_files(download_queue, processing_queue, retrieval_settings):
...
         )
         with log.log(logger):
-            input_file, clean_up = input_file.get()
+            try:
+                input_file, clean_up = input_file.get()
+            except Exception:
+                logger.exception(
+                    "Downloading of input file '%s' failed.",
+                    input_file.filename
+                )
```
Comment (adriaat) [inline ccic/bin/process.py:324]: I suggest:
```python
except Exception as e:
    logger.exception(
        "Downloading of input file '%s' failed.",
        input_file.filename,
        exc_info=e
    )
```
to no throw away any information we can have on why the download failed.

**Review by adriaat — COMMENTED** [review]
I'm not sure if the PR was ready to be 'reviewed', but I think so form our chat. If you read my suggestion, and whether you want to incorporate it or not, feel free to bypass the merge rules if you don't want to wait another day for it to be merged.

File: ccic/bin/process.py (hunk 1 above)
Comment (simonpf) [reply ccic/bin/process.py:324]: I checked it, and it seems that adding exc_info does not add any information. The traceback is plotted even if it is omitted.

**Review by adriaat — APPROVED** [review]
Good work 👍

**Pull request merged**